tests/unit/prediction/ocr/test_category_from_AOC.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Optional
+
+import pytest
+
+from robotoff.prediction.ocr.category_from_AOC import AOC_REGEX, find_category_from_AOC
+
+
+@pytest.mark.parametrize(
+    "text,value_tags",
+    [
+        ("Appellation Clairette de Die Controlée", ["fr:clairette-de-die"]),
+        ("Appellation Clairette de Die Protégée", ["fr:clairette-de-die"]),
+        ("Chinon appellation d'origine protégée", ["fr:chinon"]),
```

Comment:
Add checks on spaces:

```suggestion
        ("Chinon appellation d'origine protégée", ["fr:chinon"]),
        ("Chinon    appellation\t d'origine\n protégée", ["fr:chinon"]),
```